### DISCUSSION TITLE
Prearm: Main loop slow (168Hz<400Hz) on new version V4.5.3

### DISCUSSION THREAD

**Discussion by thiensu1122** [HefnySco/OBAL#53] [Trouble Shooting]
Hi Hefny,

Today I tried to build the new version V4.5.3, and I got the error `Prearm: Main loop slow (168Hz<400Hz)`. 
![image](https://github.com/HefnySco/OBAL/assets/20256382/dbdbd9de-2de7-43c7-8905-7dd9b2128d81)
I also noticed that the CPU usage is significantly lower than V4.2.3. 
![image](https://github.com/HefnySco/OBAL/assets/20256382/5d75bf8e-d722-4b6f-8709-e856eaccf2c0)

Do you know how to handle this?
Thank you very much,

**Comment by HefnySco** [comment]
What is the RPI you are using?
Also please send me your settings to replicate it here.

**Comment by HefnySco** [comment]
Also please try to run ardupilot as a command line not a service and see
what happens.

**Comment by thiensu1122** [comment]
Thank you for your quick reply Hefny.

I've tried to run the arducopter without using the arducopter.service but still the problem persists. Then I just turn it off for a while and then turn it on, and now the issue is no more. I have no idea how or why the problem solve itself. 

Best regards,
